### ISSUE TITLE
Add Capacitor v5 support

### ISSUE THREAD

**Issue by Plastiquewind** [Aashu-Dubey/capacitor-statusbar-safe-area#7]
Hi!
Could you please update your plugin to make it compatible with Capacitor v5?
Thanks in advance.

**Comment by Aashu-Dubey** [comment]
Hi, 

I've created a [new branch](https://github.com/Aashu-Dubey/capacitor-statusbar-safe-area/tree/chore/cap-v5-support) that adds support for Capacitor v5, based on whatever recommended changes I could find on their [official doc](https://capacitorjs.com/docs/next/updating/plugins/5-0) and [create-plugin](https://github.com/ionic-team/create-capacitor-plugin/) repo, but since it's still in beta, I'm not gonna merge or publish it till it's stable, as it may required more changes.

Till then, if you want to use the plugin in your project using Capacitor v5, you can temporarily try the branch `chore/cap-v5-support` by adding this line in you `package.json`
```json
"@aashu-dubey/capacitor-statusbar-safe-area": "git+https://github.com/Aashu-Dubey/capacitor-statusbar-safe-area.git#chore/cap-v5-support"
```

Do give it a try and let me know if it works for you.

Thanks!

**Comment by Plastiquewind** [comment]
Unfortunately it doesn't work because in this case no js bundles appear in `node_modules/@aashu-dubey/capacitor-statusbar-safe-area`. Currently I'm using a [forked](https://github.com/Plastiquewind/capacitor-statusbar-safe-area) version which includes the `dist` folder with all compiled js/ts files and it seems working.

**Comment by natterstefan** [comment]
Hi @Aashu-Dubey,

do you have any news on this issue? Thanks for your help and work!

**Comment by Aashu-Dubey** [comment]
released [v2.1.0](https://www.npmjs.com/package/@aashu-dubey/capacitor-statusbar-safe-area/v/2.1.0) with Capacitor v5 support
